spec/factories/users.rb

```diff
@@ -1,5 +1,6 @@
-require 'bcrypt'
+require 'bcrypt' # We don't have this in our gemfile
 
+# We have way to many different users here. Should have a few test users depending on roles? User, Admin, etc.
```

Comment:
As comment says, we don't need all these factories... Needs to be way more simpler then this. Lets name them smarter (`jill` doesn't explain what the user can do) and you only need many 3-4 factories for the different user permission levels (have maybe two for a register user, an admin user, and maybe like ccid/legacy user?)